index.html

```diff
@@ -1020,10 +1020,8 @@ <h2>Resource Authorization</h2>
       </p>
 
       <blockquote class="informative">
-        The [[!SOLIDWEBAC]] specification provides refinement on the original
-        <a href='https://www.w3.org/wiki/WebAccessControl'>Web Access Control wiki</a>. This section both adds further
-        refinement by clarifying (MUST, SHOULD, MAY) requirements where [[!SOLIDWEBAC]] offers open guidance, as well
-        as by specifying interactions relative to resources defined by [[!LDP]].
+        This section both adds further refinement by clarifying (MUST, SHOULD, MAY) requirements where [[!SOLIDWEBAC]]
```

Comment:
Now the wiki comment is deleted I don't think we can have "further", but perhaps the start could be made simpler anyway. How about:

> This section both clarifies (MUST, SHOULD, MAY) requirements where [[!SOLIDWEBAC]] ...